src/components/Nav/Nav.tsx

```diff
@@ -44,15 +48,16 @@ function Nav({ loggedUser, logout }: NavProps) {
           <NavItem>contact</NavItem>
         </div>
         <div className={`user ${openUserProfile ? 'open' : ''}`}>
-          {loggedUser ? (
+          {user ? (
             <div
               className="user__avatar"
               role="button"
               onClick={() => setOpenUserProfile(!openUserProfile)}
               onKeyDown={() => setOpenUserProfile(!openUserProfile)}
               tabIndex={0}
+              aria-label="User profile"
             >
-              <img src={`${user?.avatar ? user.avatar : loggedUser?.photoURL}`} alt="" />
+              <img src={`${userData?.avatar ? userData.avatar : user?.photoURL}`} alt="" />
```

Comment:
A to nie będzie tak, że jeśli user nie ma fotki, to src będzie undefined? Czyli w pewnych sytuacjach fotki w ogóle nie będzie i wygląd strony się posypie?

Reply:
Przy rejestracji uzytkownika jest nadawany mu placeholder avatar czyli to photoURL wiec jesli nie ustawi uzytkownik avatara samodzielnie to bedzie wyswietlany placeholder